mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -1,18 +1,15 @@
 import numpy as np
 import netCDF4
-from netCDF4 import Dataset as netcdf_dataset
-import xarray as xr
 import pandas as pd
 import datetime
 
-from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, \
-    remove_repeated_time_index
-
 from ..shared.plot.plotting import timeseries_analysis_plot
 
 from ..shared.io import NameList, StreamsFile
 from ..shared.io.utility import buildConfigFullPath
 
+from ..shared.MpasAnalysisXarray.MpasAnalysisXarray import openMultifileDataSet
```

Comment:
@xylar, I think this organization is strange.  Do we *really* need a folder with the same class name?  Also, typically files with classes are not camel-cased, but would be something like `from ...shared.mpas_analysis_xarray import OpenMultifileDataSet` where I'm assuming `OpenMultifileDataSet` is a class.  If it is a function it should be lower case with `_` characters.